src/aio/native_linux_aio_provider.cpp

```diff
@@ -68,14 +70,33 @@ error_code native_linux_aio_provider::flush(dsn_handle_t fh)
 error_code native_linux_aio_provider::write(const aio_context &aio_ctx,
                                             /*out*/ uint32_t *processed_bytes)
 {
-    ssize_t ret = pwrite(static_cast<int>((ssize_t)aio_ctx.file),
-                         aio_ctx.buffer,
-                         aio_ctx.buffer_size,
-                         aio_ctx.file_offset);
-    if (ret < 0) {
-        return ERR_FILE_OPERATION_FAILED;
+    int retries = 2;
```

Comment:
Why we only try it twice here? What if the write operation still fail ?

Reply:
It will core dump in the following code if  still fail. I don't make sure it will suceess if we let it try forever, so if retry failed, I choose this write is failed.